site/src/component/AppHeader/Profile.scss

```diff
@@ -76,83 +75,37 @@
     height: 50px;
     border-radius: 100%;
   }
+}
 
-  button {
-    border: none;
-    background: none;
-    font-size: 20px;
-    line-height: 1em;
-    padding: 0 0.5em;
-    border-radius: var(--border-radius);
-    display: flex;
-    align-items: center;
-    gap: 8px;
-    span {
-      font-size: 18px;
-    }
+.profile-popover__theme-section {
+  .MuiDivider-root {
+    margin: 4px 0 0;
   }
 
-  button:hover {
-    background-color: var(--overlay2);
+  h4 {
+    font-size: 20px;
   }
 }
 
 .profile-popover__links {
-  ul {
-    list-style: none;
+  .MuiListItem-root {
     padding: 0;
-    margin: 0;
-    font-size: 16px;
   }
 
-  li div {
-    display: flex;
-    gap: 4px;
-    align-items: center;
-  }
-  li svg.bi {
-    width: 18px;
-    height: 18px;
-  }
-
-  a {
-    color: var(--text);
-  }
+  .MuiListItemButton-root {
+    padding: 10px;
+    border-radius: var(--border-radius);
 
-  button {
-    border: none;
-    background: none;
-    width: 100%;
+    &:hover {
+      text-decoration: underline;
+    }
   }
-}
-
-.profile-popover__link,
-.theme-popover__link {
-  display: flex;
-  gap: 10px;
-  padding: 10px;
-  border-radius: var(--border-radius);
 
-  &:hover {
-    background-color: var(--overlay2);
+  .MuiListItemIcon-root {
+    min-width: 32px;
   }
 }
 
-.profile-popover__link.active,
-.theme-popover__link.active {
-  color: var(--blue-primary);
-}
```

Comment:
Because this was removed, navigating to that page no longer highlights it in the profile menu. I imagine this behavior is not something we want to get rid of